python/ray/rllib/examples/multiagent_pendulum.py

```diff
@@ -46,10 +46,11 @@ def create_env(env_config):
     config["horizon"] = horizon
     config["use_gae"] = True
     config["model"].update({"fcnet_hiddens": [256, 256]})
-    options = {"multiagent_obs_shapes": [3, 3],
-               "multiagent_act_shapes": [1, 1],
-               "multiagent_shared_model": True,
-               "multiagent_fcnet_hiddens": [[32, 32]] * 2}
+    options = {"multiagent_obs_shapes": [3, 3, 3],
+               "multiagent_act_shapes": [1, 1, 1],
+               "is_shared_model": True,
+               "shared_model_list": [2,1], # defines which agents share a model i.e first two share a model
```

Comment:
Would it make sense for this to be a list of model indices instead, e.g. [0, 0, 1] instead of [2, 1]?

Reply:
Oh I see, that would also work. I thought it might be more intuitive to list how many agents you have correspond to each model

Reply:
I was thinking it would be more consistent with multiagent_obs_shape and multiagent_act_shape above

Reply:
Makes sense, will change.